201-aci-linux-multiple-container-groups/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
you don't have any parameters defined in the template, so you can't pass anything to the deployment